### PR TITLE
Audit check box

### PR DESCRIPTION
**Pull request by GraceGardner**
## Description & motivation 💭 
Add colors to checkbox. 

### Design Considerations 🎨 
Need's Ash's approval 

## Testing 🧪 
Storybook
### How was this tested 👻 

- [ ] Manual testing
- [ ] E2E tests added
- [ ] Unit tests added

### Issue(s) closed 
https://temporalio.atlassian.net/browse/DT-1751

### CODE REVIEW

File: src/lib/holocene/checkbox.svelte [hunk 1]
```diff
@@ -118,11 +118,17 @@
...
   }
 
   .checkbox.hoverable:hover .checkmark::before {
-    @apply absolute -left-2.5 -z-10 h-9 w-9 self-center rounded-full bg-interactive/20 content-[''];
+    @apply absolute -left-2.5 -z-10 h-9 w-9 self-center content-[''];
+  }
+
+  .checkbox.checked .checkmark {
+    &::after {
+      @apply bg-pink;
```
Comment (laurakwhit) [inline src/lib/holocene/checkbox.svelte:130]: Is this doing anything? Look like maybe it was left in by accident.

File: src/lib/holocene/checkbox.svelte [hunk 2]
```diff
@@ -137,8 +143,12 @@
     @apply outline outline-interactive;
   }
 
+  input:checked ~ .checkmark {
+    @apply border-interactive bg-interactive;
+  }
+
   .checkmark {
-    @apply relative box-content flex h-4 w-4 flex-none cursor-pointer rounded-sm border border-primary bg-white dark:bg-transparent;
+    @apply relative box-content flex h-4 w-4 flex-none cursor-pointer rounded-md border-2 border-subtle bg-transparent text-white hover:border-white/30 hover:bg-interactive  hover:shadow-[inset_0_0_0_1.5px] hover:shadow-white hover:dark:border-black/30 hover:dark:shadow-black;
```
Comment (laurakwhit) [inline src/lib/holocene/checkbox.svelte:151]: I think we can make this match the designs a little better by utilizing `outline` ⬇️ 

```suggestion
    @apply relative box-content flex h-4 w-4 flex-none cursor-pointer rounded-md border-2 border-subtle bg-transparent text-white outline outline-4 outline-transparent hover:border-transparent hover:border-white hover:bg-interactive hover:outline-[*insert color here*] dark:text-black hover:dark:border-black;
```

File: src/lib/holocene/checkbox.svelte [hunk 3]
```diff
@@ -137,8 +143,12 @@
     @apply outline outline-interactive;
   }
 
+  input:checked ~ .checkmark {
+    @apply border-interactive bg-interactive;
```
Comment (laurakwhit) [inline src/lib/holocene/checkbox.svelte:147]:
```suggestion
    @apply border-interactive bg-interactive hover:border-white hover:dark:border-black;
```

**Review by laurakwhit — CHANGES_REQUESTED** [review]
Looks like we need to audit the following possible states for the checkbox as well
* disabled
* valid/error
* hoverable (currently, if this is set to `false` the hover styles are still applied)

We also might want to confirm the styles for `:focus-visible` (when the checkbox is tabbed to) are what we want ⬇️ 

Screenshot 2024-04-05 at 10 23 55 AM

File: src/lib/holocene/checkbox.svelte (hunk 1 above)
Comment (GraceGardner) [reply src/lib/holocene/checkbox.svelte:130]: Nice catch! Thanks Laura!

**Comment by GraceGardner** [comment]
> Looks like we need to audit the following possible states for the checkbox as well
> 
> * disabled
> * valid/error
> * hoverable (currently, if this is set to `false` the hover styles are still applied)
> 
> We also might want to confirm the styles for `:focus-visible` (when the checkbox is tabbed to) are what we want ⬇️
> 
> Screenshot 2024-04-05 at 10 23 55 AM

I'll get this done and added to this PR. Thanks Laura! I appreciate all your help!

**Comment by laurakwhit** [comment]
|Hover (dark) |Hover (light)| Disabled (dark) | Disabled (light) | Error (dark) | Error (light) |
|-|-|-|-|-|-|
|Screenshot 2024-04-15 at 11 36 14 AM|Screenshot 2024-04-15 at 11 36 04 AM|Screenshot 2024-04-15 at 11 36 41 AM|Screenshot 2024-04-15 at 11 36 49 AM|Screenshot 2024-04-15 at 11 37 48 AM|Screenshot 2024-04-15 at 11 37 56 AM|
|Screenshot 2024-04-15 at 11 35 10 AM|Screenshot 2024-04-15 at 11 36 04 AM|Screenshot 2024-04-15 at 11 37 15 AM|Screenshot 2024-04-15 at 11 37 05 AM|Screenshot 2024-04-15 at 11 37 42 AM|Screenshot 2024-04-15 at 11 38 03 AM|
|Screenshot 2024-04-15 at 11 35 20 AM|Screenshot 2024-04-15 at 11 35 49 AM|
|Screenshot 2024-04-15 at 11 35 26 AM|Screenshot 2024-04-15 at 11 35 40 AM|

**Review by rossedfort — APPROVED** [review]

**Review by laurakwhit — APPROVED** [review]

**Pull request merged**